app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class DashboardController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    account = current_user.account
+    render json: {
+      today_amount: calc_payment_amount_of(account, Date.today),
+      tomorrow_amount: calc_payment_amount_of(account, Date.tomorrow),
+      chargeback_refunds: calc_chargeback_and_refunds(account),
+      month: generate_month(account)
+    }, status: 200
+  end
+
+  private
+
+  def calc_payment_amount_of(account, date)
+    establishments = Establishment.where(shop: Shop.where(account: account))
```

Comment:
This could be an instance @variable to avoid calculating it again.
You could do something like

```
def establishments
  @establishments ||= Establishment.where(shop: Shop.where(account: account))
end
```

and call it like `establishments`.
Check the `||=` operator